src/rational/q/default.rs

```diff
@@ -111,6 +111,81 @@ impl Q {
             den: fmpz(842468587426513207),
         },
     };
+
+    /// Returns an instantiation of [`Q`] with value `2^62 - 1`.
+    ///
+    /// # Examples
+    /// ```
+    /// use qfall_math::rational::Q;
+    ///  
+    /// let a: Q = Q::MAX62;
+    /// ```
+    pub const MAX62: Q = Q {
+        value: fmpq {
+            num: fmpz(i64::pow(2, 62) - 1),
+            den: fmpz(1),
+        },
+    };
+
+    /// Returns an instantiation of [`Q`] with value `1 / (2^62 - 1)`.
+    ///
+    /// # Examples
+    /// ```
+    /// use qfall_math::rational::Q;
+    ///  
+    /// let a: Q = Q::INV_MAX62;
+    /// ```
+    pub const INV_MAX62: Q = Q {
+        value: fmpq {
+            num: fmpz(1),
+            den: fmpz(i64::pow(2, 62) - 1),
+        },
+    };
+
+    /// Returns an instantiation of [`Q`] with value `1 / (2^32 - 1)`.
+    ///
+    /// # Examples
+    /// ```
+    /// use qfall_math::rational::Q;
+    ///  
+    /// let a: Q = Q::INV_MAX32;
+    /// ```
+    pub const INV_MAX32: Q = Q {
+        value: fmpq {
+            num: fmpz(1),
+            den: fmpz(i64::pow(2, 32) - 1),
+        },
+    };
+
+    /// Returns an instantiation of [`Q`] with value `1 / (2^16 - 1)`.
+    ///
+    /// # Examples
+    /// ```
+    /// use qfall_math::rational::Q;
+    ///  
+    /// let a: Q = Q::INV_MAX16;
+    /// ```
+    pub const INV_MAX16: Q = Q {
+        value: fmpq {
+            num: fmpz(1),
+            den: fmpz(i64::pow(2, 16) - 1),
+        },
+    };
+
+    /// Returns an instantiation of [`Q`] with value `1 / (2^8 - 1)`.
+    ///
+    /// # Examples
+    /// ```
+    /// use qfall_math::rational::Q;
+    ///  
+    /// let a: Q = Q::INV_MAX8;
+    /// ```
+    pub const INV_MAX8: Q = Q {
+        value: fmpq {
+            num: fmpz(1),
+            den: fmpz(i64::pow(2, 8) - 1),
+        },
+    };
```

Comment:
If we add the inverse, I would also add the non-inverse - otherwise this might seem strange to users.